utils/warnings.ml

```diff
@@ -304,7 +304,8 @@ let () = parse_options false defaults_w;;
 let () = parse_options true defaults_warn_error;;
 
 let message = function
-  | Comment_start -> "this is the start of a comment."
+  | Comment_start -> "this is the start of a comment.\n\
+                      Hint: Did you mean `( * )'?"
```

Comment:
Would it not be a bit clearer to precise that `( * )` is an operator:  `` Did you mean the operator `( * )'? `` ?

Reply:
Good point!